### DISCUSSION TITLE
Project64 for new 3s?

### DISCUSSION THREAD

**Discussion by Dragon3166** [project64/project64#2306] [Ideas]
This gets made for new 3ds I will happy to contribute for this project play games on new 3ds with daedalus emulator I can play some n64 games on new 3ds but project64 does it better with no problems if this gets made for new 3ds I will have no problem paying 15 -30 for this project play n64 games on new 3ds if it is possible for creator to do.,

I really enjoy it for windows

**Comment by DerekTurtleRoe** [comment]
@Dragon3166 Porting an entire emulator to a game console is a lot of work, and right now we are focusing on improving the emulator itself and porting it to Android. If a developer comes along and wants to help with a port, that would be fine, but unless that happens I doubt we will port it to 3DS.
